Automattic/newspack-custom-content-migrator
# Fix/existing images captions

## How to test
- Run the newspack-content-migrator hcn-migrate-images-from-json command
- Notice the success messages for imported credit and captions for existing images. 

---
